src/main/java/ratpack/zipkin/RequestAnnotationExtractor.java

```diff
@@ -16,14 +16,18 @@
 package ratpack.zipkin;
 
 import com.github.kristofa.brave.KeyValueAnnotation;
+import java.util.Arrays;
 import ratpack.http.Request;
 
 import java.util.Collection;
 import java.util.Collections;
+import zipkin.TraceKeys;
 
 @FunctionalInterface
 public interface RequestAnnotationExtractor {
   Collection<KeyValueAnnotation> annotationsForRequest(Request request);
 
-  RequestAnnotationExtractor DEFAULT = (request -> Collections.emptyList());
+  RequestAnnotationExtractor DEFAULT = (request -> Collections.singletonList(
+      KeyValueAnnotation.create(TraceKeys.HTTP_URL, request.getPath())));
```

Comment:
Thanks for flagging this.  I need to take a closer look at the result of `request.getPath()` from a running app to ensure this is the right key to use.  Something I have notice from the Brave Servlet filter is that the `http.url` annotation is actual a path and not the entire URL. I'm not suggesting we should duplicate that here though :)